<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QDialog" name="HistoryWindow">
 <property name="styleSheet">
 <string notr="true">
 /* ОКНО ИСТОРИИ - ДОПОЛНИТЕЛЬНЫЕ СТИЛИ */
 #historyList::item {
     background-color: #F9F9FF;
     border-left: 4px solid #E8D4F1;
 }

 #filterButton {
     background-color: #D4F1E8;
 }

 #clearFilterButton {
     background-color: #F2D4E8;
 }

 QCheckBox {
     spacing: 8px;
     font-size: 11pt;
 }

 QCheckBox::indicator {
     width: 18px;
     height: 18px;
     border-radius: 4px;
     border: 1px solid #D0D0D0;
 }

 QCheckBox::indicator:checked {
     background-color: #D4E8F2;
     border-color: #B0C4CE;
 }
 </string>
 </property>
  <property name="windowTitle">
   <string>История покупок</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="dateCheckBox">
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="categoryCheckBox">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="categoryCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterButton">
       <property name="text">
        <string>Фильтровать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearFilterButton">
       <property name="text">
        <string>Сбросить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="historyList"/>
   </item>
   <item>
    <widget class="QLabel" name="countLabel">
     <property name="text">
      <string>Найдено записей: 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
